studio/src/components/cache/cache-warmer-config.tsx

```diff
@@ -0,0 +1,148 @@
+  const user = useUser();
+  const namespace = router.query.namespace as string;
+  const { mutate: configureCacheWarmerConfig, isPending } =
+    useMutation(configureCacheWarmer);
+  const [maxOperationsCount, setMaxOperationsCount] = useState(
+    currentOperationsCount.toString(),
+  );
+  const { toast } = useToast();
+
+  const operationsCountOptions = [
+    { value: "100", label: "100" },
+    { value: "200", label: "200" },
+    { value: "300", label: "300" },
+    { value: "400", label: "400" },
+    { value: "500", label: "500" },
+    { value: "1000", label: "1000" },
+  ];
+
+  return (
+    <Card>
+      <CardHeader>
+        <div className="flex w-full items-center justify-between">
+          <div className="flex flex-col gap-y-2">
+            <CardTitle>Cache Warmer Configuration</CardTitle>
+            <CardDescription className="text-sm text-muted-foreground">
+              {cacheWarmerEnabled
+                ? "Configure the cache warmer configuration of this namespace."
+                : "Enable cache warmer to set the configuration."}
+            </CardDescription>
+          </div>
+          <Button
+            type="submit"
+            variant="default"
+            isLoading={isPending}
+            disabled={
+              !cacheWarmerEnabled ||
+              !checkUserAccess({
+                rolesToBe: ["admin", "developer"],
+                userRoles: user?.currentOrganization.roles || [],
+              })
+            }
+            onClick={() => {
+              configureCacheWarmerConfig(
+                {
+                  namespace,
+                  enableCacheWarmer: cacheWarmerEnabled,
+                  maxOperationsCount: parseInt(maxOperationsCount),
+                },
+                {
+                  onSuccess: (d) => {
+                    if (d.response?.code === EnumStatusCode.OK) {
+                      toast({
+                        description: "Cache warmer config set successfully.",
+                        duration: 3000,
+                      });
+                    } else if (d.response?.details) {
+                      toast({
+                        description: d.response.details,
+                        duration: 3000,
+                      });
+                    }
+                    refetch();
+                  },
+                  onError: (error) => {
+                    toast({
+                      description:
+                        "Could not set the cache warmer config. Please try again.",
+                      duration: 3000,
+                    });
+                  },
+                },
+              );
+            }}
+          >
+            Apply
+          </Button>
+        </div>
+      </CardHeader>
+      <CardContent>
+        <div className="flex w-full flex-col gap-y-3">
+          <div className="flex w-full flex-col justify-between gap-y-4 rounded-md border p-4 md:flex-row md:items-center">
```

Comment:
I find this box in box layout quite confusing. Any idea to make it a bit more appealing?